mongo_db_conection.py

```diff
@@ -0,0 +1,422 @@
+#Document information
+document_data = {
```

Comment:
No need to add the document data to each page, just a setup py like https://github.com/bitfinexcom/bitfinex-api-py/blob/master/setup.py is good.